packages/fast-usdc/src/util/bank.js

```diff
@@ -0,0 +1,14 @@
+/* global globalThis */
+
+export const queryUSDCBalance = async (
```

Comment:
please annotate the types

Reply:
Done

Reply:
I'm getting used to this style vs `@param`. More like `.ts` and will make automatic conversions to that syntax easier.